controller/controller_server/csi_controller_server.py

```diff
@@ -534,6 +539,76 @@ def GetCapacity(self, request, context):
         logger.info("finished GetCapacity")
         return csi_pb2.GetCapacityResponse()
 
+    def ControllerExpandVolume(self, request, context):
+        set_current_thread_name(request.volume_id)
+        logger.info("ControllerExpandVolume")
+        secrets = request.secrets
+
+        try:
+            utils.validate_expand_volume_request(request)
+
+            user, password, array_addresses = utils.get_array_connection_info_from_secret(secrets)
+
+            try:
+                array_type, volume_id = utils.get_volume_id_info(request.volume_id)
+            except ObjectIdError as ex:
+                logger.warning("volume id is invalid. error : {}".format(ex))
+                context.set_code(grpc.StatusCode.INVALID_ARGUMENT)
+                return csi_pb2.ControllerExpandVolumeResponse()
+
+            with get_agent(user, password, array_addresses, array_type).get_mediator() as array_mediator:
+                logger.debug(array_mediator)
+
+                size = request.capacity_range.required_bytes
+
+                if size > array_mediator.maximal_volume_size_in_bytes or \
+                        size < array_mediator.minimal_volume_size_in_bytes:
```

Comment:
style suggestion:
```
min_size = array_mediator.minimal_volume_size_in_bytes
max_size = array_mediator.maximal_volume_size_in_bytes
if not min_size <= size <= max_size:
   ...
```
but only if you think it's better

Reply:
it is looks better
done